### ISSUE TITLE
Tolerance Issue

### ISSUE THREAD

**Issue by yunjoonjung-PNNL** [pnnl/ANIMATE#34]
Hello Team!

I just found that we've used tolerance inconsistently. Some verification items include ```tol``` inside the ```verify``` method and other items defined ```tol``` in ```new_library_verification_cases.json``` file. Which one is the recommended way for this?

**Comment by leijerry888** [comment]
Good catch. To me, this is question for verification algorithm design. For the same verification item, do we want tolerance to be potentially different for different verification cases to different buildings / systems? Is the tolerance part of the knowledge, or a parameter that is customizable by the user to accommodate different data sources etc. For different items, the answer may be different.
